src/config/v2.rs

```diff
@@ -60,19 +56,11 @@ pub(crate) fn from_config_string(config_string: &str) -> Result<InstanceConfig,
                 None => (config_toml, ""),
             };
 
-            static RX_DIRECTIVE: Lazy<Regex> =
-                Lazy::new(|| Regex::new(r#"^[A-Za-z0-9_-]+$"#).expect("directive regex"));
-
             if !RX_DIRECTIVE.is_match(directive) {
-                return Err(format!("'{directive}' is not a valid directive or TOML key-value pair.\n\n{original_error}"));
+                return Err(format_invalid_directive(directive, original_error));
             }
 
-            let mut config: UserInput = match toml::from_str(config_toml) {
-                Ok(config) => config,
-                Err(error) => {
-                    return Err(format!("TOML parsing error: {error}"));
-                }
-            };
+            let mut config = user_input_from_config_toml(dbg!(config_toml))?;
```

Comment:
Hey @tommilligan, seems like this line now causes quite some prints during the build that are not really relevant. Very minor issue, but maybe that was not intended ?

Looks like this: 
`.cargo/registry/src/index.crates.io-6f17d22bba15001f/mdbook-admonish-1.17.0/src/config/v2.rs:63:58] config_toml = "collapsible=true \ntitle=\"Display value (without control)\""`

Reply:
Thanks, yes this was indeed unintentional. Opened #186 to resolve